README.md

```diff
@@ -39,8 +52,18 @@ Earth Enterprise Fusion & Server currently run on 64-bit versions of the followi
 * Ubuntu 14.04 LTS
```

Comment:
Let's remove Ubuntu 14 from list of supported OS by Open GEE 5.3.2 https://github.com/google/earthenterprise/blob/release_5.3.2/README.md